app/main.py

```diff
@@ -1,3 +1,66 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: object | int) -> object:
```

Comment:
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```

Reply:
change everywhere

Reply:
![Screenshot 2024-05-23 at 12 29 33](https://github.com/mate-academy/py-distance-class/assets/105336389/a7319b75-044c-4f10-b99e-d999caeb7a82)
i try to do it, but catch the error:  F821 undefined name 'Distance'